docs/001_develop/03_client-capabilities/022_framework-integration/002_integration-angular.mdx

```diff
@@ -0,0 +1,279 @@
+    ></foundation-form>
+  `,
+  standalone: true,
+  schemas: [CUSTOM_ELEMENTS_SCHEMA],
+})
+export class AppComponent {
+  resourceName = 'EVENT_INSERT_USER';
+}
+```
+
+### Boolean
+
+To set a boolean attribute on a component you can either hard code it in the template or use Angular template bindings `[]`.
+
+```ts
+import { Component, CUSTOM_ELEMENTS_SCHEMA } from '@angular/core';
+
+@Component({
+  selector: 'my-root',
+  template: `
+    <rapid-button [disabled]="isDisabled">
+      Disabled button
+    </rapid-button>
+    <!-- or hard code it -->
+    <rapid-button disabled>
+      Disabled button
+    </rapid-button>
+  `,
+  standalone: true,
+  schemas: [CUSTOM_ELEMENTS_SCHEMA],
+})
+export class AppComponent {
+  isDisabled = true;
+}
+```
+
+### Number
+
+To set a text attribute on a component, you can either hard code it in the template or use Angular template bindings `[]`.
+
+```ts
+import { Component, CUSTOM_ELEMENTS_SCHEMA } from '@angular/core';
+
+@Component({
+  selector: 'my-root',
+  template: `
+    <rapid-grid-pro>
+      <grid-pro-server-side-datasource
+        resource-name="ALL_TRADES"
+        [maxRows]="maxRows"
+      ></grid-pro-server-side-datasource>
+    </rapid-grid-pro>
+    <!-- or hard code it -->
+     <rapid-grid-pro>
+      <grid-pro-server-side-datasource
+        resource-name="ALL_TRADES"
+        max-rows="100"
+      ></grid-pro-server-side-datasource>
+    </rapid-grid-pro>
+  `,
+  standalone: true,
+  schemas: [CUSTOM_ELEMENTS_SCHEMA],
+})
+export class AppComponent {
+  maxRows = 100;
+}
+```
+
+## Setting properties
+
+Properties are part of the Javascript object representation of the DOM element. They are used for dynamic interaction and setting complex non-primitive data types such as objects and arrays.
+
+You can set these using Angular template bindings `[]`.
+
+```ts
+import { Component, CUSTOM_ELEMENTS_SCHEMA } from '@angular/core';
+
+@Component({
+  selector: 'my-root',
+  template: `
+    <foundation-form
+      design-system-prefix="rapid"
+      [jsonSchema]="jsonSchema"
+    ></foundation-form>
+  `,
+  standalone: true,
+  schemas: [CUSTOM_ELEMENTS_SCHEMA],
+})
+export class AppComponent {
+  jsonSchema = {
+      type: 'object',
+      properties: {
+        QUANTITY: {
+          type: 'number',
+          description: 'kotlin.Double',
+        },
+        SIDE: {
+          type: 'string',
+          description: 'kotlin.String',
+        },
+      },
+  };
+}
+```
+
+## Handling events
+
+You can handle events from Foundation UI components using Angular's event binding `()`.
+
+```ts
+import { Component, CUSTOM_ELEMENTS_SCHEMA } from '@angular/core';
+
+@Component({
+  selector: 'my-root',
+  template: `
+    <foundation-form
+      (submit)="handleSubmit($event)"
+    ></foundation-form>
+  `,
+  standalone: true,
+  schemas: [CUSTOM_ELEMENTS_SCHEMA],
+})
+export class AppComponent {
+  handleEvent(e): void {
+    console.log('form submitted', e);
+  }
+}
+```
+

```

Comment:
I know you're going to elabroate on the styling parts later, that's fine. But are we able to talk about slots here please too?

Reply:
Added slots